src/iterateJsdoc.js

```diff
@@ -136,6 +136,10 @@ const getUtils = (
       handler();
       const replacement = utils.stringify(jsdoc, specRewire);
 
+      if (!replacement) {
+        return fixer.removeRange(jsdocNode.range);
```

Comment:
I changed this in 5c6e9ac42351fe549a310a34894a76cc1ab13fa5 because the `0` index was relative to the source file.

Reply:
Ouch, yeah. I've added f4459a56b0d46bdc7fb62be70239ecc9b52116fc to restore the whitespace removal. How does that look?

Reply:
Oop sorry forgot to reply here - looks good, thanks!